### PR TITLE
make release tool aware of the enterprise repos

### PR DESCRIPTION
**Pull request by mchalski**
https://tracker.mender.io/browse/MEN-1611

@kacf your review is the most important here.
I think I filled in every section of the map, not sure about the `release_component` bool flag - should it be true or false?
also - will the minimal enterprise compose layer do the job?

### CODE REVIEW

File: extra/release_tool.py [hunk 1]
```diff
@@ -256,8 +266,8 @@ class Component:
                 "release_component": False,
             },
             "mender-useradm": {
-                "git": ["useradm"],
-                "docker_image": ["useradm"],
+                "git": ["useradm", "useradm-enterprise"],
+                "docker_image": ["useradm", "useradm-enterprise"],
                 "release_component": True,
```
Comment (merlin-northern) [inline extra/release_tool.py]: why here under mender-useradm key you put useradm-enterprise?

Comment (mchalski) [reply extra/release_tool.py]: I have to defer to @kacf here.
this was just a semi-intelligent copy-paste based on other components.

Comment (merlin-northern) [reply extra/release_tool.py]: just as mine yesterday :) https://github.com/mendersoftware/integration/pull/640

Comment (kacf) [reply extra/release_tool.py]: It's because the `useradm-enterprise` image is a *replacement* for the `mender-useradm` container, not an extra container.

**Pull request merged**